#object center position 

geometry_msgs/Vector3 goal_position
geometry_msgs/Quaternion goal_orientation
string pfield_id
---
bool success